internal/cli/internal/command/open/open.go

```diff
@@ -0,0 +1,66 @@
+package open
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/skratchdot/open-golang/open"
+	"github.com/spf13/cobra"
```

Comment:
typically one separates 3rd party imports before their own, as per the rest of the docs.